### ISSUE TITLE
Oi

### ISSUE THREAD

**Issue by sivaljunior** [ggabriel96/pamonhinha#2]
Uma classe pra mexer com Conjuntos. Deem uma olhada no UML e avisem se precisar de mais alguma coisa

**Comment by vladimirbelinski** [comment]
Implementem algo com números primos e com nomes de pessoas.

**Comment by vladimirbelinski** [comment]
Vocês receberam alguma dica? Repassem...

**Comment by sivaljunior** [comment]
Dicas: "quando é o aniversário da pessoa mais velha?" e "quantas semanas desde meu ultimo almoço"

**Comment by sivaljunior** [comment]
Nosso Conjunto vai ser um ArrayList de Objects, pq pode ser um conjunto de strings tbm. E os retornos são os objetos Double, Integer e tal, pq se n acha nada retorna null

**Comment by acaciaterra** [comment]
Vladimir, vocês só tem uma dica?

**Comment by sivaljunior** [comment]
Ele passou algo de dica pra vcs?

**Comment by jpwbernardi** [comment]
Cade o UML de vocês?

**Comment by vladimirbelinski** [comment]
As dicas foram: "talvez números primos?" e "ou talvez nomes de pessoas?"

**Comment by vladimirbelinski** [comment]
É um arquivo .java o UML

**Comment by jpwbernardi** [comment]
Ah =P

**Comment by vladimirbelinski** [comment]
Nosso UML é uma classe para trabalhar com Datas.

**Comment by acaciaterra** [comment]
O nosso, com conjuntos

**Comment by vladimirbelinski** [comment]
Implementem algo pensando em um conjunto que pode conter nomes de pessoas... acho que não só dados numéricos, pois ele fala de pessoas e primos.

**Comment by jpwbernardi** [comment]
Mas é object por isso

**Comment by sivaljunior** [comment]
Sim, fizemos um ArrayList de Objects
Aí pode ter strings e tals

**Comment by vladimirbelinski** [comment]
Ok :) Deem push!

**Comment by acaciaterra** [comment]
Já foi

**Comment by sivaljunior** [comment]
Dica: "20 minutos é muito pouco!! Preciso de mais!"

**Comment by vladimirbelinski** [comment]
GRITEM PANQUECA QUE GANHA TEMPO!

**Comment by acaciaterra** [comment]
NÃÃÃÃÃO

**Comment by sivaljunior** [comment]
É? AUHEHUEAHU
Sério isso?

**Comment by vladimirbelinski** [comment]
GRITEM PANQUECA + PAMONHINHA

**Comment by vladimirbelinski** [comment]
GRITEM PARA O FERNANDO

**Comment by vladimirbelinski** [comment]
SIM, SÉRIO KKKK

**Comment by acaciaterra** [comment]
FOI

**Comment by sivaljunior** [comment]
Foi

**Comment by jpwbernardi** [comment]
Começa já a codar ou não?

**Comment by sivaljunior** [comment]
A gente começa a codar já??

**Comment by vladimirbelinski** [comment]
Não recebemos orientações, mas acho que podem sim, seria bom...

**Comment by acaciaterra** [comment]
Vocês estão codando?

**Comment by ggabriel96** [comment]
Gente, não sei o que fazer no UML em relação a essa "quando é o aniversário da pessoa mais velha?". Querem que coloque um atributo que seja uma classe Pessoa?

**Comment by ggabriel96** [comment]
Ainda não codamos nada, e vocês?

**Comment by sivaljunior** [comment]
Não codamos nada ainda

**Comment by vladimirbelinski** [comment]
Vamos iniciar a implementar agora, ele está passando o problema...

**Comment by sivaljunior** [comment]
Um método que mexe com primos, no nosso, é meio desnecessário, pq não faz nem sentido ter isso na classe Conjunto

**Comment by vladimirbelinski** [comment]
Comecem a implementar a classe também.

**Comment by acaciaterra** [comment]
Ta.

**Comment by jpwbernardi** [comment]
Por que tem nome?

**Comment by vladimirbelinski** [comment]
Pensamos que as dicas de nome eram para nós. Só depois percebemos que eram para vocês...

**Comment by jpwbernardi** [comment]
Ah, ok

**Comment by acaciaterra** [comment]
Sobre isso aqui: public Data(int i, boolean feriado) { // se quiser setar só um dos valores
Como vou saber qual valor? Se é dia, mês ou ano?

**Comment by ggabriel96** [comment]
Qualquer um, esse é o ponto. Talvez tu só esteja interessado em dia, ou mês, ou ano... daí só coloquei ali. Sei lá XD

**Comment by acaciaterra** [comment]
TA, MAS COMO VOU SABER?

**Comment by acaciaterra** [comment]
HSUAHSUAHSUAH

**Comment by ggabriel96** [comment]
ACÁCIA, É UM CONSTRUTOR

**Comment by acaciaterra** [comment]
Eu sei

**Comment by jpwbernardi** [comment]
Mas não tem nada dizendo se é dia, mes ou ano

**Comment by sivaljunior** [comment]
Gente, pergunta. Nos métodos de conversão, tipo, se eu tenho 12/05/2013, eu faço 12 + 5_30 + 2013_365?

**Comment by sivaljunior** [comment]
Tem um vezes ali HUAEHUAE

**Comment by ggabriel96** [comment]
Sim, só que perde o sentido ter três construtores com 1 int como parâmetro, só mudando o nome do argumento. Acho que nem compila, inclusive

**Comment by vladimirbelinski** [comment]
Pessoal precisamos que a implementação de vocês nos permita realizar o seguinte: fazer um programa que receba 4 datas do usuário (horários de aula). O programa informa qual é o encontro mais próximo do sábado.

**Comment by sivaljunior** [comment]
A gente precisa disso: Programa que lê N números, e imprime de trás pra frente, utilizando apenas a classe Conjuntos

**Comment by sivaljunior** [comment]
Dica
"Onde diz sábado, na verdade é segunda, onde diz 4, na verdade é 3"

**Comment by vladimirbelinski** [comment]
Ok

**Comment by vladimirbelinski** [comment]
Dica para vocês: " Onde diz números na verdade são palavras. Onde diz N na verdade é 10".

**Comment by sivaljunior** [comment]
Blz

**Comment by ggabriel96** [comment]
Dá pra alterar o UML? Se colocarem lá, eu faço um método que imprime de trás pra frente ou algo do tipo

**Comment by vladimirbelinski** [comment]
Pessoal, disponibilizem o código assim que implementarem a classe.

**Comment by ggabriel96** [comment]
Fernando disse que não pode alterar o UML. Mas dá pra imprimir de trás pra frente com o que tem lá

**Comment by jpwbernardi** [comment]
Já fiz o código que precisavamos

**Comment by ggabriel96** [comment]
Ah blz

**Comment by vladimirbelinski** [comment]
Implementaram a nossa classe já?

**Comment by sivaljunior** [comment]
Quase

**Comment by jpwbernardi** [comment]
Eles tão fazendo e ta dando falha de segmentação quando tento adicionar uma string

**Comment by jpwbernardi** [comment]
Vocês não inicializam valores

**Comment by jpwbernardi** [comment]
tem que instanciar o objeto

**Comment by ggabriel96** [comment]
Sim, mas não tem construtor

**Comment by jpwbernardi** [comment]
Faz no add então

**Comment by jpwbernardi** [comment]
Se não foi instanciado, instancia

**Comment by sivaljunior** [comment]
Hey
Mudou tudo

**Comment by vladimirbelinski** [comment]
O importante é diferença das datas. Implementem essa primeiro.

**Comment by sivaljunior** [comment]
Não é mais segunda, é terça

**Comment by ggabriel96** [comment]
Ah é, nem me toquei ahuhuahua já vai

**Comment by sivaljunior** [comment]
E não é 3, é 10

**Comment by sivaljunior** [comment]
E onde diz próximo, é longe

**Comment by ggabriel96** [comment]
Deem pull

**Comment by vladimirbelinski** [comment]
Para vocês:
- não é mais palavras, são floats...
- não é 10, é 11...
- onde diz  "trás para frente" é "maior que PI de trás para frente"

**Comment by jpwbernardi** [comment]
Sim

**Comment by vladimirbelinski** [comment]
Para vocês:
- não é mais floats, são inteiros pares...

**Comment by sivaljunior** [comment]
Pra vcs:
Não é mais terça, é 1 do próximo mês

**Comment by jpwbernardi** [comment]
SE O FERNANDO MUDAR DE NOVO EU VOU SURTAR

**Comment by ggabriel96** [comment]
HUAHUAHUAHUA SIM PQP

**Comment by sivaljunior** [comment]
GENTE, PRA DIFERENÇA DE DATA TEM BILHÕES DE CASOS

**Comment by ggabriel96** [comment]
Sim, huahuahua tava falando com o Vladimir agora. Façam levando em conta dia e mês só

**Comment by acaciaterra** [comment]
Será?

**Comment by sivaljunior** [comment]
Ta

**Comment by acaciaterra** [comment]
Ah, vai HSUAHSUAHSUAH

**Comment by sivaljunior** [comment]
E com horas, minutos e segundos? Fazer diferença tbm?

**Comment by vladimirbelinski** [comment]
Foquem na diferença do dia e mês. Já repasso um problema para vocês.

**Comment by jpwbernardi** [comment]
Gente, corrigi bug

**Comment by acaciaterra** [comment]
Passa aí

**Comment by jpwbernardi** [comment]
O fernando ta passando alguma coisa ai?

**Comment by ggabriel96** [comment]
Passou, Vladimir já manda

**Comment by acaciaterra** [comment]
Tarefa pra vocês fazerem: Faça um programa que receba 5 datas (com dia, mes e ano). O programa informa quantas datas são maiores e quantas são menores que a data formada pela média de todas as datas.

**Comment by acaciaterra** [comment]
Exemplo: 1/3/2000
29/5/1000 
Média: 15/4/1500
Maiores: 1
Menores: 1

**Comment by vladimirbelinski** [comment]
Tarefa para vocês:

Faça um programa que receba 5 números float. O programa deve colocar os números menores do que o médio deles no conjunto "menores". Os maiores que o médio vão no conjunto "maiores". Imprimir os conjuntos na forma:

Menores: a, b, c
Maiores: d, e

a ... e são números

**Comment by ggabriel96** [comment]
Tá, então agora precisa da diferença levando em conta dia, mês e ano, se quiserem fazer AHUAHUHUAHUAHUA

**Comment by acaciaterra** [comment]
HSUAHSUAHSUHAUSHAUSHUAHSUAHS

**Comment by sivaljunior** [comment]
Ta, faço ano, perai

**Comment by ggabriel96** [comment]
Gente, Data.java não compila

**Comment by acaciaterra** [comment]
Onde que da erro?

**Comment by ggabriel96** [comment]
Tem umas bostas do GitHub lá no meio, fora os protótipos do UML de antes

**Comment by acaciaterra** [comment]
To arrumando, calma ai

**Comment by sivaljunior** [comment]
Vê se foi

**Comment by ggabriel96** [comment]
Acho que ainda não, tá igual o arquivo

**Comment by ggabriel96** [comment]
Gente, eu preciso mexer com datas "menores", iguais ou que vem depois... Façam aquele método de diferença retornar uma Data com os atributos negativos caso a data venha antes, tudo 0 se for igual ou positivos se vier depois. Tem como?

**Comment by sivaljunior** [comment]
AI MEUDEUS
Ta, espera

**Comment by acaciaterra** [comment]
GABRIEL

**Comment by sivaljunior** [comment]
hEY, NÃO SAO 5 DATAS, SÃO 7
E as menores nao precisa mais fazer

**Comment by acaciaterra** [comment]
AGORA COMPILA, TINHA UM MONTE DE LIXO LÁ

**Comment by ggabriel96** [comment]
Tá, então não precisa do negócio negativo lá

**Comment by sivaljunior** [comment]
COMMITA TUDO PQ VAI ACABAR

**Comment by vladimirbelinski** [comment]
Suponham que <<<< >>>> seja o texto modificado:

Faça um programa que receba <<<<10>>>> números float. O programa deve colocar os números menores do que o médio  deles no conjunto "menores". Os maiores que o médio  <<<< e menores que 10>>>> vão no conjunto "maiores". Imprimir os conjuntos na forma:

Menores: a, b, c
Maiores: d, e
<<<< Excluídos: f >>>>

a ... e são números
<<<< f: números maiores que 10 e maiores que a média >>>>

**Comment by sivaljunior** [comment]
AFFE GABRIEL, JA FIZ A PORRA NEGATIVO

**Comment by vladimirbelinski** [comment]
Ok

**Comment by ggabriel96** [comment]
Tá, mas o Fernando que mudou né ahuahuahua

**Comment by sivaljunior** [comment]
Sim, mas agora foi negativo u.u

**Comment by ggabriel96** [comment]
Deixa

**Comment by sivaljunior** [comment]
Eu vou deixar ahsuehuae

**Comment by acaciaterra** [comment]
Isso, deixa shuahsuas

**Comment by sivaljunior** [comment]
COMMITA TUDO

**Comment by sivaljunior** [comment]
PQ TA ACABANDO

**Comment by acaciaterra** [comment]
GENTE, EU DESCOBRI QUE EU SOU A MARLI 832

**Comment by vladimirbelinski** [comment]
ELE VAI DESLIGAR OS PCS EM 1 MIN!!!!!

**Comment by sivaljunior** [comment]
VAI DESLIGAAR
